server/api/repositories/block.repository.ts

```diff
@@ -90,7 +90,7 @@ export class BlockRepository implements IBlockRepository {
             return true;
         } catch (error) {}
     }
-    async findBlockedByEmailOrUsername(
+    async findBlockedByUserIdAndEmailOrUsername(
         query: string,
         userId: string,
         offset: number = 0,
```

Comment:
Nếu query = "" thì lúc filter bị sai logic vì like = "" sẽ trả về toàn bộ danh sách người bị chặn --> đã có userId rồi thì chỉ cần lọc where dựa vào userId này th không cần phải so với email hay username làm gì. Còn phần filterBy chắc chỉ lọc theo updatedAt thôi.

Reply:
cái này filter "" thì nó trả về toàn bộ danh sách chặn của user đó mà ông, sai gì hả ông, với ko cần search theo email với username đúng ko ông, tại tui thấy bữa web kìa ông cho xem có ô search chỗ block nên tui mới thêm theo email với username, ko cần thì lát tui update bỏ